packages/plugins/response-cache/src/plugin.ts

```diff
@@ -209,6 +209,13 @@ const originalDocumentMap = new WeakMap<DocumentNode, DocumentNode>();
 const addTypeNameToDocument = memoize1(function addTypeNameToDocument(
   document: DocumentNode,
 ): DocumentNode {
+  if (
+    document.definitions.some(
+      def => def.kind === Kind.OPERATION_DEFINITION && def.operation === 'subscription',
```

Comment:
What if a document contains multiple subscriptions but a query is selected?

Reply:
I didn't though this was even a thing ><

How does it works ? It will both run the execution and subscription in parallel ? 

I should then probably just filter definitions and modify only queries and mutations.

Reply:
Ok, I think i've understood how it's supposed to work. I had to modify the test kit to be able to give it the operation name.